common/src/main/java/com/revenuecat/purchases/common/errors.kt

```diff
@@ -40,7 +41,11 @@ enum class BackendErrorCode(val value: Int) {
 fun Exception.toPurchasesError(): PurchasesError {
     return when (this) {
         is JSONException, is IOException -> {
-            PurchasesError(PurchasesErrorCode.NetworkError, localizedMessage)
+            if (this is UnknownHostException) {
```

Comment:
i put this `if` within the `when` case because UnknownHostException is a kind of IOException. We could check it before IOException, but we'd probably want to add a comment to maintain that order...

Reply:
Oh I see. I think that's normal to have more specific exceptions first anyway.

Reply:
yeah you're right, will update